app/views/pages/homepage.html.erb

```diff
@@ -1 +1,84 @@
-<h1>Homepage</h1>
+<div class="usa-grid homepage-hero">
+  <h2>While moving is never easy, <b>move.mil</b> makes your experience just a little bit better.</h2>
+</div>
+
+<div class="usa-grid">
+  <aside class="usa-width-one-third">
+    <div class="dps-login-section">
+      <p><b>Defense Personal Property System (DPS)</b> is your portal to move scheduling and the claims process.</p>
+      <p><%= link_to raw("Log In to #{abbr_tag('dps')}"), 'https://eta.sddc.army.mil/ETASSOPortal/SSO/AppLogin.aspx?LoginType=portal&amp;SSOAppID=DPS', class: 'usa-button' %></p>
+      <p class="usa-text-small">New to <%= abbr_tag('dps') %>? <%= link_to 'Create your account', 'https://www.move.mil/dod/first_time_users/browser_compatibility.cfm' %></p>
+    </div>
+  </aside>
+
+  <div class="usa-width-two-thirds">
+    <h2 class="homepage-content-title">Your Moving Guide</h2>
+
+    <div class="homepage-section usa-media_block">
+      <%= image_tag 'homepage/plan-your-move.jpg', alt: '', class: 'usa-media_block-img' %>
+      <div class="usa-media_block-body">
+        <h3>Plan Your Move</h3>
+        <p>It’s never too early to get started, learn what you are allowed to bring and plan ahead.</p>
+
+        <ul>
+          <li><%= link_to 'Learn about Moving Allowances (Entitlements)', page_path('entitlements') %></li>
+          <li><%= link_to 'Verify Service-Specific Information', '#TODO' %></li>
```

Comment:
_Verify_ Service-Specific Information? Is this supposed to point to the service specific information page or something else?